qtism/data/state/ExternalScored.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * @license GPLv2
+ *
+ * @author Bartłomiej Marszał <bartlomiej@taotesting.com>
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+namespace qtism\data\state;
+
+use qtism\common\enums\Enumeration;
+
+class ExternalScored implements Enumeration
```

Comment:
After check it seems that of enumeration constant are integer, should we keep this logic? because currently `getConstantByName` seems useless => `getConstantByName('human') == 'human'`

Reply:
I agree. I will remove this method?

Reply:
After the conversation, it seems that the only neede change is to modify the lines 31 and 33. All the parsing is still working.